### PR TITLE
Implements LogFetcher and LogFetcher::Papertrail

### PR DESCRIPTION
**Pull request by joker1007**
I implemented `LogFetcher` to simplify checking command output.

### CODE REVIEW

**Comment by joker1007** [comment]
Sample

```
% be wrapbox ecs run_cmd -f config/wrapbox.yml -c repro-development --timeout=7200000 --cpu=1024 --memory=2048 "bundle exec rake -T"
D, [2017-09-12T20:38:30.241806 #13421] DEBUG -- : Container Definitions: [dummy]
D, [2017-09-12T20:38:30.628224 #13421] DEBUG -- : Task Options: {dummy}
D, [2017-09-12T20:38:30.710412 #13421] DEBUG -- : Create Task: dummy
D, [2017-09-12T20:38:35.771336 #13421] DEBUG -- : Launch Task: dummy
Sep 12 20:38:43 54.199.249.17 wrapbox-default: rake about                                                                                   # List versions of all Rails frameworks and the environment

# .. omit ..

Sep 12 20:38:43 54.199.249.17 wrapbox-default: rake tmp:clear                                                                               # Clear session, cache, and socket files from tmp/ (narrow w/ tmp:sessions:clear, tmp:cache:clear, tmp:sockets:clear)
Sep 12 20:38:43 54.199.249.17 wrapbox-default: rake tmp:create                                                                              # Creates tmp directories for sessions, cache, sockets, and pids
D, [2017-09-12T20:38:45.898394 #13421] DEBUG -- : Stop Task: dummy

**Comment by joker1007** [comment]
@kei-q @abicky plz review

File: lib/wrapbox/log_fetcher/papertrail.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          @query = search['query']
+        end
+
+        @query ||= ''
+
+        search_query = ::Papertrail::SearchQuery.new(connection, @query, query_options)
+
+        until @stop do
+          search_query.next_results_page.events.each do |event|
+            next if event.received_at < @started_at
```
Comment (abicky) [inline lib/wrapbox/log_fetcher/papertrail.rb]: `received_at` supports timezone ✅

**Comment by joker1007** [comment]
It is tiresome for me that re-implement papertrail API client 😅 
I am worry about same point too.
But papertrail-cli repository is not active.
And even if this feature is broken temporalily, user find out probrem soon and omit this feature easily.
Because of it, It is not big probrem.

**Pull request merged**